core/server/master/src/main/java/alluxio/master/file/activesync/ActiveSyncManager.java

```diff
@@ -446,4 +446,19 @@ public void setTxId(long mountId, long txId) {
   public ExecutorService getExecutor() {
     return mExecutorService;
   }
+
+  /**
+   * Stop the sync manager.
+   */
+  public void stop() {
+    for (long mountId : mFilterMap.keySet()) {
+      try {
+        stopSyncForMount(mountId);
+      } catch (IOException e) {
+        LOG.info("IOException encountered in stopping activeSyncManager {}", e);
```

Comment:
for both of these, can we log the mount id and the mount URI?

Reply:
mountURI involves mount table look up, logged mount id for now.